driver/src/price_estimation/dexag.rs

```diff
@@ -341,7 +352,7 @@ mod tests {
 
         let client = DexagClient::new(&HttpFactory::default()).unwrap();
         let before = Instant::now();
-        let prices = client.get_prices(tokens).unwrap();
+        let prices = client.get_prices(tokens).wait().unwrap();
```

Comment:
Does `now_or_never` not work in this case?

Reply:
This is the online dexag client so the future is not going to be guaranteed ready.

Reply:
:see_no_evil: